script.js

```diff
@@ -83,6 +83,7 @@ function writeMilliSecondToDoc(givenMilliSecondTime, writeToDiv) {
 let elapsedTime = 0;
 let hasPausedStopWatch = true;
 let timerIntervalStopWatchId = null;
+let stopwatchStarted = false;
```

Comment:
Instead of creating a new variable, why don't you use the existing `hasPausedStopWatch` variable?